pkg/operator/operator.go

```diff
@@ -120,29 +122,36 @@ func NewOperator() (context.Context, *Operator) {
 			return ctx
 		},
 	})
-	manager = lo.Must(manager, err, "failed to setup manager")
+	mgr = lo.Must(mgr, err, "failed to setup manager")
 	if opts.EnableProfiling {
-		registerPprof(manager)
+		registerPprof(mgr)
 	}
-	lo.Must0(manager.GetFieldIndexer().IndexField(ctx, &v1.Pod{}, "spec.nodeName", func(o client.Object) []string {
+	lo.Must0(mgr.GetFieldIndexer().IndexField(ctx, &v1.Pod{}, "spec.nodeName", func(o client.Object) []string {
 		return []string{o.(*v1.Pod).Spec.NodeName}
 	}), "failed to setup pod indexer")
 
 	return ctx, &Operator{
-		Manager:             manager,
```

Comment:
Why are we hiding this now? Seems like we have to pass most of the fields over anyways.